pkg/internal/checkup/executor/oslat/client.go

```diff
@@ -0,0 +1,200 @@
+/*
+ * This file is part of the kiagnose project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package oslat
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	expect "github.com/google/goexpect"
+
+	"github.com/kiagnose/kubevirt-realtime-checkup/pkg/internal/checkup/executor/console"
+)
+
+type consoleExpecter interface {
+	SafeExpectBatchWithResponse(expected []expect.Batcher, timeout time.Duration) ([]expect.BatchRes, error)
+}
+
+type Client struct {
+	consoleExpecter consoleExpecter
+	testDuration    time.Duration
+}
+
+func NewTest(vmiUnderTestConsoleExpecter consoleExpecter, testDuration time.Duration) *Client {
```

Comment:
The function returns a `Client` object.
```suggestion
func NewClient(vmiUnderTestConsoleExpecter consoleExpecter, testDuration time.Duration) *Client {
```

Reply:
DONE